SparkPost/2web2ui
# SA-1827: UX feedback for report builder

### What Changed

- Bounce reason table includes count and percent now
- Date picker limits to 6 months

### How To Test

- Go to report builder
- Open date picker and verify that date picker doesn't allow for dates before 6 months before today
- Close date picker and add any bounce metrics
- Go to bounce tab
- Verify that count and percent are now included in value

### To Do
